src/containers/ResourceForm/ResourceForm.test.jsx

```diff
@@ -124,6 +124,18 @@ describe('The ResourceForm component', () => {
           instance.setVisibilityOfFields(formData);
           expect(wrap.state().uiResource['ams:serviceType']['ui:widget']).toBe('select');
         });
+
+        it('should hide byteSize when ams:distributionType is api', () => {
+          const formData = { 'ams:distributionType': 'api' };
+          instance.setVisibilityOfFields(formData);
+          expect(wrap.state().uiResource['dcat:byteSize']['ui:widget']).toBe('hidden');
+        });
+
+        it('should hide byteSize when ams:distributionType is web', () => {
+          const formData = { 'ams:distributionType': 'web' };
+          instance.setVisibilityOfFields(formData);
+          expect(wrap.state().uiResource['dcat:byteSize']['ui:widget']).toBe('hidden');
+        });
```

Comment:
Is de coverage nog hetzelfde na deze changes? Het lijkt me dat je niet alle branches gecovered hebt.

Reply:
Bijgewerkt: Coverages van branches is iets gestegen naar 64%